modules/shared/src/main/scala/org/tessellation/schema/transaction.scala

```diff
@@ -23,7 +27,9 @@ object transaction {
 
   @derive(decoder, encoder, eqv, show)
   @newtype
-  case class TransactionOrdinal(value: NonNegBigInt)
+  case class TransactionOrdinal(value: NonNegBigInt) {
```

Comment:
I think it's safe to use `NonNegLong`. I don't think it's possible for ordinal to exceed `Long` max value in any foreseeable future.

Reply:
Yep it's most probably safe, though I don't know if there is a big value in changing to `NonNegLong`. Anyway transaction schema is outside of the scope of this PR, so I can create a ticket to think about and change if needed.

Reply:
I would use the smallest types that will do the job.